openvino_tensorflow/ie_basic_engine.cc

```diff
@@ -30,59 +31,96 @@ void IE_Basic_Engine::infer(
     std::vector<std::string>& param_names) {
   load_network();
   if (m_infer_reqs.empty()) {
-    m_infer_reqs.push_back(m_exe_network.CreateInferRequest());
+    m_infer_reqs.push_back(m_compiled_model.create_infer_request());
   }
 
   //  Prepare input blobs
-  auto func = m_network.getFunction();
-  auto parameters = func->get_parameters();
+  auto parameters = m_model->get_parameters();
   for (int i = 0; i < inputs.size(); i++) {
     if (inputs[i] != nullptr) {
 #if defined(OPENVINO_2021_2)
```

Comment:
Should this part be removed? as the code would no more be compatible with earlier OV versions